assets/package.json

```diff
@@ -3,13 +3,12 @@
   "version": "0.1.0",
   "description": "The Phoenix LiveView JavaScript client.",
   "license": "MIT",
-  "main": "./priv/static/_live_view.js",
```

Comment:
let's just leave this in there for now so we can leave the smallest diff possible.